watertap/ui/tests/test_api.py

```diff
@@ -12,6 +12,7 @@
 # from pyomo.environ import units as pyunits
```

Comment:
Need to add in the copyright statement at the top of this file. (see other comments above)